internal/jsonapi/manifest/wrapper.go

```diff
@@ -12,17 +12,18 @@ import (
 
 const wrapperTemplate = `#!/bin/sh
 
+export PATH="$PATH:$HOME/.nix-profile/bin" # required for Nix
```

Comment:
I think we should wrap it into a `if [ -d <DIR> ];` statement to only update the PATH if the destination does exist. wdyt?

Reply:
Hmm, that makes sense, but would duplicate all the paths, unless we put them in variables, etc.. Having non-existing paths in `$PATH` seems to have no negative effects, so I kinda like the shorter code like this.

Reply:
I don't mind duplicating them. But you can also put them into variables if you prefer.

Reply:
I've checked and there's no problem with having non-existent paths in `PATH` so I think the simpler code is the better code. We should leave it as-is (and only add the Nix path and move them up so `gpg-agent` is found.)